### PR TITLE
data: jegrami

### PR DESCRIPTION
**Pull request by jegrami**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7466"></a>

### CODE REVIEW

File: data/jegrami.json [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
   "name": "Jeremiah Igrami",
-  "bio": "Copyeditor turned technical writer and enthusiastic programmer. I'm passionate about building solutions and telling stories with data",
+  "bio": "**Technical Writer** and **Data Scientist**. I love uncovering insights from huge, messy data.",
+  "tags": [
+    "Python",
+    "pandas",
+    "numpy",
+    "matplotlib",
+    "plotly.express",
```
Comment (jaydeepkhatri) [inline data/jegrami.json:9]: Hi @jegrami ,

Remove the `,` (comma) at the end of the tags array. JSON is invalid due to that.

```suggestion
    "plotly.express"
```

You can test your code here -> https://linkfree.io/playground

Thank you :)

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
As you're Editing your recently merged profile, you might forgot to update your branch which causes merge conflicts.

Please update your branch to resolve conflicts. (Refer these docs for help - https://linkfree.io/docs/how-to-guides/editing#edit-your-profile, *especially point no. 4* )

**Review by SaraJaoude — APPROVED** [review]

File: data/jegrami.json [hunk 2]
```diff
@@ -1,6 +1,13 @@
 {
   "name": "Jeremiah Igrami",
-  "bio": "Copyeditor turned technical writer and enthusiastic programmer. I'm passionate about building solutions and telling stories with data",
+  "bio": "**Technical Writer** and **Data Scientist**. I love uncovering insights from huge, messy data.",
```
Comment (CBID2) [inline data/jegrami.json:3]: [Recommendation] I think it’d be great to add some of the statements you had about being a technical writer in your old bio. It’ll highlight both of your interests.
```suggestion
  "bio": "**Technical Writer** and **Data Scientist**. I love uncovering insights from huge, messy data.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @jegram! Aside from your bio needing improvement, you are off to a great start! :)

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to resolve the conflicts I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.io/docs/quickstart

Editing your Profile: https://linkfree.io/docs/how-to-guides/editing

**Pull request closed without merging**